src/networkcloud/azext_networkcloud/aaz/latest/networkcloud/baremetalmachine/_replace.py

```diff
@@ -109,6 +109,9 @@ def _build_arguments_schema(cls, *args, **kwargs):
             fmt=AAZStrArgFormat(
                 min_length=1,
             ),
+            blank=AAZPromptInput(
```

Comment:
The `password` property is not `AAZPasswordArg` type. Maybe you should set its format as `password` in swagger like this https://github.com/Azure/azure-rest-api-specs/blob/6914c95db58c8dd037faf07bcc21475fc19c57f3/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/CommonDefinitions.json#L2533-L2534

Reply:
Thanks Kai , we will address this change in the next preview API's version, our current version of API are approved